packages/generator/archiveFontPackage.ts

```diff
@@ -0,0 +1,31 @@
+import fsExtra from 'fs-extra';
+import path from 'path';
+
+import {
+  ArchivedDataPath,
+  FontPackagesDir,
+  getPackageNameForWebfont,
+  ProjectRootDir,
+} from './shared';
+import { FontItem } from './types';
+
+const FontArchiveDir = path.join(ProjectRootDir, 'font-archive');
+
+export async function archiveFontPackage(webfont: FontItem) {
+  await fsExtra.ensureDir(FontArchiveDir);
+
+  const dirName = getPackageNameForWebfont(webfont);
+  if (!(await fsExtra.exists(path.join(FontPackagesDir, dirName)))) {
+    console.warn(`Font package ${webfont.family} not found at ${dirName}`);
+    return;
+  }
+  if (await fsExtra.exists(path.join(FontArchiveDir, dirName))) {
+    console.warn(`Font package ${webfont.family} already archived at ${dirName}`);
+    return;
+  }
```

Comment:
This should also make sure that the `/font-packages/webfont` package gets deleted or we'll have the package in both